.github/package-filters/rs-packages.yml

```diff
@@ -42,3 +42,8 @@ drive-abci:
   - .github/workflows/tests*
   - packages/rs-drive-abci/**
   - *drive
+
+rs-sdk:
+  - .github/workflows/tests*
+  - packages/rs-sdk/**
+  - *drive
```

Comment:
It uses drive as a dependency?

Reply:
yes, proof verification depends on drive